ci/travis/before_install_linux

```diff
@@ -2,5 +2,5 @@
 set -e
 
 sudo apt-get update
-sudo apt-get install libmatio-dev cppcheck 
+sudo apt-get install libmatio-dev cppcheck gcc-arm-none-eabi libnewlib-arm-none-eabi
```

Comment:
should we try the hard-float compilers?
i.e. ` gcc-arm-none-eabihf` ?
also, maybe later, it would be interesting to also add a `aarch64` [i.e. ARM64] build run